docker/docker-compose-build.yml

```diff
@@ -5,9 +5,6 @@ services:
   candlepin-base:
     build: candlepin-base/
     image: ${REGISTRY}/candlepin-base
-  candlepin-rhel7:
-    build: candlepin-rhel7/
-    image: ${REGISTRY}/candlepin-rhel7
```

Comment:
If we're removing this from the compose, we should also remove the whole docker/candlepin-rhel7 directory